api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatient.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+
+/**
+ * Defines a MpiPatient in the system. A mpi patient is simply an extension of a patient and all that that
+ * implies.
+ */
+public class MpiPatient extends Patient {
+
+    private static final boolean mpiPatient = true;
+
+    /**
+     * Checks if patients is MpiPatient
+     * @return true if it is mpi patient, otherwise false
+     */
+    public boolean getMpiPatient() {
```

Comment:
Yep, according JavaBean convention.
But we can't name it like that, because we use this method in "RegistrationApp" module, and there we use "simplify" for converting from Java object to JSON. And that "simplify" can't understand "is" before field name.
blob/master/omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/MatchingPatientsFragmentController.java#L93